weatherreport.cpp

```diff
@@ -67,12 +80,14 @@ void TestRainy() {
 void TestHighPrecipitationAndLowWindspeed() {
     // This instance of stub needs to be different-
     // to give high precipitation (>60) and low wind-speed (<50)
-    SensorStub sensor;
+    SensorStub sensor(72, 70, 26, 49);
 
     // strengthen the assert to expose the bug
     // (function returns Sunny day, it should predict rain)
     string report = Report(sensor);
     assert(report.length() > 0);
+    assert(report.find("Sunny") == string::npos);
```

Comment:
Good asserts :)
Here it actually reflects the user's need - users need to see 'rain' in the forecast. It can be 'rainy' or 'rain predicted'... both are equally good. And "Sunny" should not appear, of course. That would confuse the user